bgpd/bgp_nht.c

```diff
@@ -437,8 +428,9 @@ void bgp_parse_nexthop_update(int command, vrf_id_t vrf_id)
 			 * we receive from bgp.  This is to allow us
 			 * to work with v4 routing over v6 nexthops
 			 */
-			if (peer &&
-			    CHECK_FLAG(peer->flags, PEER_FLAG_CAPABILITY_ENHE)
+			if (peer && !peer->ifp
```

Comment:
I don't understand how this check is avoiding any crash. Because peer->ifp is not really accessed any where in this loop. Am i missing something ?

Reply:
Because if the peer->ifp was set, we would try to start the RA processing when it was not correct.  This was fallout from a commit done by Donald (don't remember the commit) but in testing the 5549 over global addresses hit the crash.